jaxws-ri/rt/src/main/java/com/sun/xml/ws/model/AbstractWrapperBeanGenerator.java

```diff
@@ -212,6 +213,19 @@ protected AbstractWrapperBeanGenerator(AnnotationReader<T,C,?,M> annReader,
         return responseMembers;
     }
 
+    // When an element is of an array type, the software
+    // currently sets nillable to true regardless of the value of the
+    // related annotation.  A customer has complained that nillable="false"
+    // should be allowed for such a type.
+    //
+    // Since the current behavior was specifically placed in the code,
+    // there may be a good reason for it, and we do not want to break
+    // compatibility.
+    // Therefore, we are adding a new system property
+    // -Djaxb.allowNonNillableArray=true
+    // to implement the behavior requested by the customer.
+    static final boolean JAXB_ALLOWNONNILLABLEARRAY = getBooleanSystemProperty("jaxb.allowNonNillableArray").booleanValue();
```

Comment:
Property name should probably start with ‘com.sun.xml.ws...’. Also what is the impact of this change, if one uses MOXy instead of JAXB-RI?